src/components/PathwayCard.tsx

```diff
@@ -183,34 +186,32 @@ const PathwayCard: React.FC<PathwayCardProps> = ({
             </BadgeArray>
           </div>
         </div>
-
         <div className="mt-auto pt-3 border-t border-gray-100">
-          <div className="flex flex-col">
-            <div className="w-full mb-4">
-              <div className="flex justify-between items-center">
-                <div>
-                  <p className="text-xs text-rmigray-500">Publisher:</p>
-                  <p className="text-sm font-medium text-rmigray-800">
-                    <HighlightedText
-                      text={
-                        pathway.publication.publisher.short ||
-                        pathway.publication.publisher.full
-                      }
-                      searchTerm={searchTerm}
-                    />
-                  </p>
-                </div>
-                <div className="text-right">
-                  <p className="text-xs text-rmigray-500">Published:</p>
-                  <p className="text-sm font-medium text-rmigray-800">
-                    <HighlightedText
-                      text={pathway.publication.year}
-                      searchTerm={searchTerm}
-                    />
-                  </p>
-                </div>
-              </div>
+          <div className="flex justify-between items-center mb-2">
+            <div>
+              <p className="text-xs text-rmigray-500">Publication:</p>
+              <p className="text-sm font-medium text-rmigray-800">
+                <HighlightedText
+                  text={truncateText(
+                    pathway.publication.title.short ||
+                      pathway.publication.title.full,
+                    40, // <-- set your desired max length here
```

Comment:
NB: in future we may want to have this be based on the available space, rather than a character limit (comparable to how we're handling the number of badges in `BadgeArray`)

Reply:
Very good point!

Reply:
#585